scripts/create-licenses.sh

```diff
@@ -167,6 +167,9 @@ find vendor -type f -name LICENSE.md -execdir mv LICENSE.md LICENSE ';'
 V2_LICENSE_DIR="vendor/github.com/cpuguy83/go-md2man"
 mv ${V2_LICENSE_DIR}/v2/LICENSE ${V2_LICENSE_DIR}
 
+GO_RESTFUL_LICENSE_DIR="vendor/github.com/emicklei/go-restful"
```

Comment:
Add a comment here to explain why this is necessary

Reply:
L165 above is a general comment for handling the cleanup/edge cases since L167-168 also does a similar handling for the `go-md2man/v2` package.

Will update the comment to be more descriptive!